tests/ClientTest.php

```diff
@@ -37,6 +37,18 @@ public function testConstructorAutoEncryptionOpts(): void
         new Client(static::getUri(), [], ['autoEncryption' => $autoEncryptionOpts]);
     }
 
+    #[DoesNotPerformAssertions]
+    public function testConstructorEmptyKmsProvider(): void
```

Comment:
Would this test have failed without the above patch? If not, is anything being tested at all?

Reply:
Yes:
> MongoDB\Driver\Exception\EncryptionException: expected BSON document for field: gcp